app/assets/stylesheets/components/_links.css

```diff
@@ -0,0 +1,9 @@
+a, a:visited {
```

Comment:
**suggestion:** following up on how I organize CSS, I always have a file called `_base.css` for all "classless" styles. WDYT?